health/health.go

```diff
@@ -46,7 +46,14 @@ func (h *Health) CheckProvider(name string) (bool, error) {
 
 func (h *Health) IsSupportedProvider(t pt.Type) bool {
 	switch t {
-	case pt.RedisOnline, pt.DynamoDBOnline, pt.PostgresOffline, pt.SnowflakeOffline, pt.ClickHouseOffline, pt.SparkOffline, pt.RedshiftOffline:
+	case pt.RedisOnline,
+		pt.DynamoDBOnline,
```

Comment:
nit: can you make these inline

Reply:
I mean line up vertically

Reply:
Also I wonder if you can make it so if its not supported it doesn't show up as disconnected?

Reply:
wonder if it makes sense to make this a const somewhere at the top and have it be a list so that the code is slices.contains(t, supportedHealthCheckProviders)